recipe/meta.yaml

```diff
@@ -7,7 +7,7 @@ package:
 source:
     fn: cmocean-{{ version }}.tar.gz
     url: https://github.com/matplotlib/cmocean/archive/{{ version }}.tar.gz
-    sha256: 22e40329f5fd1c6d2136d7b32dfe988f3b8570a360b86bf9786efb2cb20c9aaa
+    sha256: 7f52753cd7cb2a8ccd532dadbce8db149088e27e3ad9b9f05ec9a30b6c12db4c
```

Comment:
Are you on a Mac? I believe that computing `sha256` is different and maybe @jjhelmus can you me remember the command. 

I just computed it on my Linux box and I got: `ccc5aa3e277f7f163ac58a2670326f48310ec5154b1c85d4fb1069d623fdfa20`